source/manual/architecture-deep-dive.html.md

```diff
@@ -0,0 +1,462 @@
+---
+owner_slack: "#govuk-developers"
+title: Architectural deep-dive of GOV.UK
+section: Applications
+type: learn
+layout: manual_layout
+parent: "/manual.html"
+last_reviewed_on: 2020-03-03
+review_in: 3 months
+---
+
+We can cover most of the GOV.UK architecture by asking ourselves three questions:
+
+1. What happens when a user visits a page on GOV.UK?
+2. What happens when a publisher hits 'Publish'?
+3. What happens when a developer deploys a change to an application?
+
+[GOV.UK]: https://www.gov.uk
+
+## What happens when a user visits a page on GOV.UK?
+
+### DNS
+
+![](images/dns.png)
+
+The browser queries a local DNS server to turn the domain name into an IP address.
+The local DNS server might be able to answer immediately from its cache. If not,
+it will query the authoritative name servers for the domain.
+
+`gov.uk.` is a country-code second-level domain or ccSLD (like `co.uk.`) and its
+authoritative servers are hosted by [Jisc]. Unusually for a ccSLD, `gov.uk` is
+also a website, and hosts the redirect from `gov.uk` to `www.gov.uk`. The records
+for these two domains are within the `gov.uk` second-level zone hosted by Jisc.
+
+GDS hosts several third-level domains under `gov.uk` on [Amazon Route53], for
```

Comment:
I've now included a link to the dns.html at the end of this section, if users wish to read up on it further 👍 but I do want the document to be a reasonably comprehensive reference in its own right, so that users don't _have_ to link off to other pages to get the full picture.
Things like our DNS resolution are unlikely to change often enough for maintenance to be an issue.